llvm/test/MC/Disassembler/AMDGPU/vinterp.txt

```diff
@@ -75,181 +75,433 @@
 # CHECK: v_interp_p2_f32 v0, v1, v2, v3 clamp wait_exp:7
 
 0x00,0x00,0x02,0xcd,0x01,0x05,0x0e,0x04
-# CHECK: v_interp_p10_f16_f32 v0, v1, v2, v3 wait_exp:0
+# CHECK-TRUE16: v_interp_p10_f16_f32 v0, v1.l, v2, v3.l wait_exp:0
+# CHECK-FAKE16: v_interp_p10_f16_f32 v0, v1, v2, v3 wait_exp:0
+
+0x00,0x00,0x02,0xcd,0x01,0x05,0x0e,0x04
```

Comment:
There seems to be a number of duplicated lines in this file, running a --unique update on this test

Reply:
removed duplicated dasm testlines